subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -10502,13 +10502,12 @@ u32_t radio_adv_enable(u16_t interval, u8_t chan_map, u8_t filter_policy,
 
 		link = mem_acquire(&_radio.link_rx_free);
 		if (!link) {
-			return BT_HCI_ERR_MEM_CAPACITY_EXCEEDED;
+			return BT_HCI_ERR_CONN_LIMIT_EXCEEDED;
```

Comment:
Please swap the order of the two mem_acquire, and use the new error code for the connection context allocation.